hack/verify-yamllint.sh

```diff
@@ -21,6 +21,14 @@ fi
 LOG=/tmp/yamllint.log
 helmPath=charts/latest/azuredisk-csi-driver/templates
 
+echo "checking yaml files num ..."
+deployDirNum=`ls deploy/*.yaml | wc -l`
+helmDirNum=`ls $helmPath/*.yaml | grep -v serviceaccount | wc -l`
+if [[ "${deployDirNum}" != "${helmDirNum}" ]]; then
+  echo "yaml file num($deployDirNum) under deploy/ not equal to num($helmDirNum) under $helmPath"
+  #exit 1
```

Comment:
enable this?

Reply:
Oh, the check has been added before. Now, I remove the duplicated one.